tests/config.py

```diff
@@ -5,5 +5,6 @@
 
 import os
 
-VALIDATOR_UI_URL = os.environ.get('FILENAME', 'http://localhost:8010')
-DIFFER_UI_URL = os.environ.get('FILENAME', 'http://localhost:8030')
+VALIDATOR_UI_URL = os.environ.get('VALIDATOR_UI_URL', 'http://localhost:8010')
```

Comment:
This variable shall be renamed into **RDF_VALIDATOR_{name of the variable}** in order to be consistent with the other variables in the project. 

Most likely a harmonisation and enforcement of naming convention shall be done to avoid surprises in the future. This can be a separate ticket to look across all out projects.

Reply:
This refers to the environment variables only, no need to change the names of variables in the config.py of each of the projects (there we need to update only the expected even variable).